### ISSUE TITLE
Single Course page

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#2388]
Try to use built in Sensei blocks and helper functions for all the sidebar content.

Note that the current implementation which uses an unordered list is not accessible, and we should use a table for tabular data instead:

<img src="https://github.com/WordPress/Learn/assets/1017872/4f53716d-be52-4582-a7e6-1a721f2b06d7" width="300">

### [Design](https://www.figma.com/design/ZKaf6u8mIHkAanluWafXAp/Learn?node-id=3473-42479&m=dev)

<img src="https://github.com/WordPress/Learn/assets/1017872/b8ebbd1c-3520-4ab8-84f3-54664749ee3c" width="400">
